code/src/components/CheckboxStyled.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+const HiddenCheckbox = styled.input.attrs({ type: 'checkbox' })`
```

Comment:
It is impressive that you used checkboxes, not an easy thing to style! Good job!